nagios/commands.cfg

```diff
@@ -1,27 +1,11 @@
 define command {
            command_name check_https
-           command_line    $USER1$/check_http -I $HOSTADDRESS$ -H $HOSTADDRESS$ -S $ARG1$
+           command_line    $USER1$/check_http -I $HOSTADDRESS$ -H $HOSTADDRESS$ -S $ARG1$ -C 60
 }
 
 define command {
                command_name check_https_authenticated
-               command_line    $USER1$/check_http -I $HOSTADDRESS$ -H $HOSTADDRESS$ -a $USER3$:$USER4$ -S $ARG1$
-}
-
-define command{
-        command_name    check_bitly_proxy_https
```

Comment:
`check_bitly_proxy_https` there are still services using this command so it isn't safe to remove.